lib/imagec/download.go

```diff
@@ -153,7 +153,9 @@ func (ldm *LayerDownloader) DownloadLayers(ctx context.Context, ic *ImageC) erro
 				ldm.registerDownload(topDownload)
 				layer.Downloading = true
 				LayerCache().Add(layer)
-
+				if err := LayerCache().Save(); err != nil {
```

Comment:
If the layer is still downloading, and the layer cache is saved, and before that layer is committed, VCH is restarted, the download progress is interrupted, where to cleanup?
Feels like the downloading layer is one in memory process. Is that right to be cached in kv store?